Formula/swift-format.rb

```diff
@@ -13,7 +13,7 @@ class SwiftFormat < Formula
     sha256 "d8f72c33efc125e2904e1bec2c8942cca75d75cf81dcab7fcf08ba124af16170" => :catalina
   end
 
-  depends_on xcode: ["11.4", :build]
+  depends_on xcode: ["12"]
```

Comment:
I removed the "build" requirement,  because the final binary does call out to the locally installed Swift toolkit